README.md

```diff
@@ -17,23 +17,23 @@ interact.
 # HoloLens Tools
 
 Download all of the developer tools from the
-[Microsoft Developer Website](http://lens.ms/Sa37sr)
+[Microsoft Developer Website](https://developer.microsoft.com/en-us/windows/mixed-reality/install_the_tools?Ocid=Hololens%20-%20Education_Social_TWITTER_HoloLens_20160330_413089651&utm_source=Direct)
```

Comment:
I think we should keep the short URL. That way we can redirect to a different site from there and not have to update this document.